### ISSUE TITLE
Sentry relay - default port in configmap is wrong

### ISSUE THREAD

**Issue by jsenzier-oxeva** [sentry-kubernetes/charts#177]
Hi,

We encountered issue with the relay connection to sentry. The connection wasn't successfully because the destination port in the relay's config map is not correct :

https://github.com/sentry-kubernetes/charts/blob/develop/sentry/templates/configmap-relay.yaml

```data:
  config.yml: |-
    relay:
      mode: managed
      upstream: "http://{{ template "sentry.fullname" . }}-web:{{ template "sentry.port" }}/"
      host: 0.0.0.0
      port: {{ template "relay.port" }}
```

By default, the value in the configmap will be :

```data:
  config.yml: |-
    relay:
      mode: managed
      upstream: "http://sentry-web:9000/"
      host: 0.0.0.0
      port: 3000
```

It can't work as the external port of sentry-web is 443 (as exposed in the service) and not 9000 (it's the internal port, as mentionned in the values.yaml file) :

``` sentry-web                           ClusterIP   10.233.20.23    <none>        443/TCP                                                          3d```

We must change the configmap value with "http://sentry-web:443" to allow relay to connect to sentry, the default value in the configmap-relay.yaml seems to be wrong.

My configuration :

- k8s version => 1.14.3
- sentry chart version => 5.1.3 (app version 20.8.0)
- nginx ingress version => 1.29.5 (app version 0.27.1)

**Comment by odoucet** [comment]
The good value to use in template file should be `.Values.service.externalPort`.
cf https://github.com/sentry-kubernetes/charts/blob/a0e79cf8c834c79c1018c02452274a6f43d5ef73/sentry/templates/service-sentry.yaml#L21

**Comment by Mokto** [comment]
Please see https://github.com/sentry-kubernetes/charts/pull/181

Thanks!

**Comment by odoucet** [comment]
> Please see #181
> 
> Thanks!

This MR seems the good one :)

**Comment by Mokto** [comment]
Fixed through 5.2.1
